CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* Add full width option to select component (PR #1012)
+
 ## 17.9.1
```

Comment:
Not your change, but I just noticed we've gone from version 17.18.x to 17.9.x ... I think we need to change the Changelog to be 17.19.x (as the version.rb file itself is correct)

Reply:
Good spot, have corrected.

Looks like it was actually my change, though. Sorry...